requirements.txt

```diff
@@ -1,5 +1,5 @@
 click == 7.1.2
-opencv-python == 4.1.2.30
+opencv-contrib-python == 4.1.2.30
```

Comment:
Why is this change made? What is the difference between contrib and the normal one?

Reply:
opencv-python contains main modules for the package.
opencv-contrib-python contains both main modules and contrib/extra modules